pyproject.toml

```
@@ -34,8 +34,8 @@ dependencies = [
     "pandas",
     "pytransform3d",
     "qtpy",
+    "qt-niu",
     "scikit-image",
-    "scipy",
     "qt-niu"
```

I think `scipy` somehow disappeared from the requirements list. Could it be added back? 

```suggestion
    "scipy"
```